api/src/db/models/user_models.py

```diff
@@ -75,6 +75,16 @@ def email(self) -> str | None:
             return self.linked_login_gov_external_user.email
         return None
 
+    @property
+    def first_name(self) -> str | None:
+        if self.profile is not None:
+            return self.profile.first_name
+
+    @property
+    def last_name(self) -> str | None:
+        if self.profile is not None:
+            return self.profile.last_name
+
```

Comment:
Very nitpicky, but could we be explicit here for the null case? 

```suggestion
    @property
    def first_name(self) -> str | None:
        if self.profile is not None:
            return self.profile.first_name
            
        return None

    @property
    def last_name(self) -> str | None:
        if self.profile is not None:
            return self.profile.last_name
            
       return None

```

Reply:
Im getting this error

```
src/db/models/user_models.py: note: In member "first_name" of class "User":
src/db/models/user_models.py:83:9: error: Statement is unreachable  [unreachable]
            return None
            ^~~~~~~~~~~
src/db/models/user_models.py: note: In member "last_name" of class "User":
src/db/models/user_models.py:90:9: error: Statement is unreachable  [unreachable]
            return None
            ^~~~~~~~~~~
Found 2 errors in 1 file (checked 498 source files)
make: *** [lint-mypy] Error 1
```